### PR TITLE
Ashwini created get and post endpoints for courses entity

### PR DESCRIPTION
**Pull request by aswinicode**

### CODE REVIEW

File: src/main/java/com/example/aprilbatchproject/entity/Courses.java
Comment (kasaniraj) [inline src/main/java/com/example/aprilbatchproject/entity/Courses.java:1]: Don't change the Entity. keep this as common.

File: src/main/java/com/example/aprilbatchproject/repository/CourseRepository.java
Comment (kasaniraj) [inline src/main/java/com/example/aprilbatchproject/repository/CourseRepository.java:1]: resolve the conflict.

File: src/main/java/com/example/aprilbatchproject/service/CourseService.java
Comment (kasaniraj) [inline src/main/java/com/example/aprilbatchproject/service/CourseService.java:1]: resolve the conflict.

**Review by jaisaha27d — COMMENTED** [review]
in course controller line 18 return courseDto in APi response

File: src/main/java/com/example/aprilbatchproject/controller/CourseController.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import java.util.List;
+
+@RestController
+@RequestMapping("/courses")
+public class CourseController {
+    @Autowired
+    CourseService courseService;
+
+    @PostMapping("/create_course")
+    public String createCourse(@RequestBody Courses courses) throws Exception {
```
[the remark this answers is not in the document]
Comment (aswinicode) [reply src/main/java/com/example/aprilbatchproject/controller/CourseController.java]: Thanks for find done.

File: src/main/java/com/example/aprilbatchproject/entity/Courses.java
Comment (aswinicode) [reply src/main/java/com/example/aprilbatchproject/entity/Courses.java:1]: ok sure.

File: src/main/java/com/example/aprilbatchproject/repository/CourseRepository.java
Comment (aswinicode) [reply src/main/java/com/example/aprilbatchproject/repository/CourseRepository.java:1]: done.

File: src/main/java/com/example/aprilbatchproject/service/CourseService.java
Comment (aswinicode) [reply src/main/java/com/example/aprilbatchproject/service/CourseService.java:1]: done

**Pull request merged**